<launch>
  <node name="prius_msg_to_cmd_vel_node" pkg="prius_msg_converter" type="prius_msg_to_cmd_vel" output="screen">
    <param name="wheel_base" value="2.7" type="double"/>
    <param name="max_steer_angle" value="0.47891" type="double"/>
    <param name="max_velocity" value="37.9983" type="double"/>
    <param name="kp_throttle" value="0.75" type="double"/>
    <param name="ki_throttle" value="0.2" type="double"/>
    <param name="kp_brake" value="0.5" type="double"/>
    <param name="ki_brake" value="0.1" type="double"/>
    <param name="publish_rate" value="50" type="double"/>
  </node>
</launch>
